tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -475,7 +475,10 @@ public function theUserSharesUsingWebUIWithoutClosingDialog(
 					__METHOD__ . " share with path '$resource' for user '$currentUser' could not be found."
 				);
 			}
-			$this->featureContext->setLastShareIdOf($currentUser, $shareId);
+			$this->featureContext->addToCreatedUserGroupShares(
+				$currentUser,
+				$this->featureContext->getResponseXml(null, __METHOD__)
+			);
```

Comment:
get `shareItem` instead of `shareId` from `foreach ($shareData as $shareItem) {` block where `shareId` is set
```suggestion
			$this->featureContext->addToCreatedUserGroupShares($currentUser, $shareItem);
```